src/elements/ProjectSettingsGeneral/ProjectSettingsGeneral.js

```diff
@@ -288,8 +290,9 @@ ProjectSettingsGeneral.propTypes = {
 }
 
 export default connect(
-  ({ projectStore }) => ({
-    projectStore
+  ({ appStore, projectStore }) => ({
+    projectStore,
+    default_artifact_path: appStore.frontendSpec?.default_artifact_path
```

Comment:
use camelCase here. I would suggest to get the entire `frontendSpec` instead of just `default_artifact_path`